packages/firestore/src/core/firestore_client.ts

```diff
@@ -146,7 +146,11 @@ export class FirestoreClient {
      * an async I/O to complete).
      */
     public asyncQueue: AsyncQueue,
-    private databaseInfo: DatabaseInfo,
+    /**
+     * @internal
```

Comment:
I don't think `@internal` is necessary here, since `FirestoreClient` is not a public API.